Sources/Shared/API/MJPEGStreamer.swift

```diff
@@ -7,12 +7,13 @@ class MJPEGStreamerSessionDelegate: SessionDelegate {
     static var taskUserInfoKey: AnyHashable = "taskUserInfoKey"
 
     // if/when alamofire also implements this again, we need to update to handle it as the breakpoint between images
-    func urlSession(
+    override func urlSession(
```

Comment:
@zacwest Based on the comment on this function, do you see a problem with this change? What needs to be checked?

Reply:
Does this still work? I think it should probably still be fine with the default implementation happening, but it's definitely necessary to manually test.

Reply:
I just tested, it's working